### DISCUSSION TITLE
Welcome to web2project Dicsussions

### DISCUSSION THREAD

**Discussion by caseysoftware** [web2project/web2project#345] [Announcements]
We’re using Discussions as a place to connect with other members of our community. We hope that you:

* Ask questions you’re wondering about.
* Share ideas.
* Engage with other community members.
* Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, start a thread and let's get going!

**Comment by JishnuMohanP** [comment]
Hi, after the configuration of web2project, I'm not able to create project and task. Please help to resolve. After clicking the create project submit button. Page is reloading and no project is getting created. It is showing some permission issue when I view the projects. I tried with admin login with sufficient permissions. Please help to resolve.
